Dockerfile

```diff
@@ -487,6 +488,7 @@ RUN ln /rootfs/sbin/init /rootfs/sbin/wrapperd
 RUN chmod +x /rootfs/sbin/wrapperd
 RUN ln /rootfs/sbin/init /rootfs/sbin/dashboard
 RUN chmod +x /rootfs/sbin/dashboard
+RUN ln -s kmod /rootfs/sbin/modprobe
```

Comment:
all `kmod` utils are symlinks to `kmod`. Eg: `lsmod`, `rmmod` etc, since we only need modprobe manually creating the symlink